mixmaxhq/mixmax-sdk-js
# Move the sequence picker button into an iframe

To prevent its styling from being overridden and to allow the UI to be updated
remotely.

Fixes https://app.asana.com/0/19141607276671/488741505511329.

To test:
1. Check out https://github.com/mixmaxhq/compose/pull/5504.
2. Edit https://github.com/mixmaxhq/mixmax-sdk-js/blob/master/src/utils/Environment.js#L24 to read compose-local.
3. `yarn start`.
4. Load http://localhost:9000/examples/sequencepicker/index.html.
